tools/NuGet/template-nuget/DynamoVisualProgramming.DynamoServices.nuspec

```diff
@@ -15,7 +15,13 @@
         It also contains useful static events and event data for monitoring workspace and execution changes.</summary>
         <copyright>Copyright Autodesk 2023</copyright>
         <dependencies>
-            <group targetFramework="netstandard2.0" />
+            <group targetFramework="netstandard2.0">
+                <dependency id="System.Reflection.MetadataLoadContext" version="8.0.0" />
```

Comment:
What are the cases where we would need this? I don't suspect net48 targeting apps would need this assembly. Can't we just exclude it all the time?

Reply:
As stated in the description, given the current APIs available in DynamoServices, and the use case for ZT authoring - I agree with your suspicion. It's not clear that those APIs will always remain internal though.

I could also see someone using the DynamoServices package to build some kind of integration rather than to author ZT nodes.

Since we do not sufficiently describe the use cases for these packages it's hard to reason about - IF someone somehow found themselves in a position where they were building a net48 app using DynamoServices and invoked these python load apis the binaries would need to be there.

We could certainly simplify it to always exclude if we're okay dealing with that later, or calling it out of the intended use case for this package - which IMO does make more sense to compile against instead of to consume.